es/getting_started/update.md

```diff
@@ -63,6 +63,6 @@ B. Se abre una ventana del editor que tengas configurado en git, mostrando el me
 
 C. Recibes mensajes de error de git junto con un `Automatic merge failed; fix conflicts and then commit the result.`. Esto significa que se han encontrado conflictos entre los cambios en tu código y los cambios que se realizaron en CONSUL desde la última vez que actualizaste tu fork. Esta es una de las principales razones para intentar mantener tu fork lo más al dia posible, realizando este proceso al menos mensualmente. Resuelve manualmente los conflictos para terminar el merge y haz un commit.
 
-Now you can just simply push **upstream** branch to github and create a Pull Request so you can easily check all changes going into your repo, and see your tests suite runs.
+Ahora simplemente pushea la rama **upstream** a github y crea un Pull Request, así podrás ver de manera sencilla todos los cambios que se han realizado en el repositorio y verás también como arrannca la suite de tests.
```

Comment:
Just so it doesn't get lost in the "resolved conversations" section, I'll add one of the comments originally on this line:

I don't think "pushea" is an actual word in Spanish thinking. How about "sube"?